src/main/java/uk/ac/ebi/ampt2d/accession/variant/VariantAccessioningRestController.java

```diff
@@ -36,16 +37,13 @@
     @Autowired
     private AccessioningService accessioningService;
 
-    @RequestMapping(value = "/getAccessions", method = RequestMethod.POST, produces = "application/json",
-            consumes =
-                    "application/json")
+    @RequestMapping(method = RequestMethod.POST, produces = "application/json", consumes = "application/json")
     public Map<VariantMessage, String> accessionVariants(@RequestBody List<VariantMessage> variantMessages) {
         return accessioningService.getAccessions(variantMessages);
     }
 
-    @RequestMapping(value = "/getVariants", method = RequestMethod.POST, produces = "application/json", consumes =
-            "application/json")
-    public Map<String, VariantMessage> getVariants(@RequestBody List<String> accessions) {
+    @RequestMapping(value = "/getVariants", method = RequestMethod.GET, produces = "application/json")
+    public Map<String, VariantMessage> getVariants(@RequestParam("accession") List<String> accessions) {
```

Comment:
I was thinking about the URL more in the lines of 

* `v1/accession/variant/rs123` (single values)
* `v1/accession/variant/rs234,rs567` (multiple values)

than `v1/accession/variant?accession=rs123,rs234`.